<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGPA Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>CGPA Calculator</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sem1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="sem2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="sem3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="sem4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="sem5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="sem6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="sem7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="sem8">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate CGPA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semester 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semester 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your CGPA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>470</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semester 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semester 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semester 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semester 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semester 7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semester 8</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCGPA_Calculator">
    <property name="title">
     <string>CGPA Calculator</string>
    </property>
   </widget>
   <addaction name="menuCGPA_Calculator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
